File: widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/RemoveScreenSharingNotificationUseCase.java
package com.glia.widgets.core.notification.domain;

import com.glia.widgets.core.notification.device.INotificationManager;

public class RemoveScreenSharingNotificationUseCase {
    private final INotificationManager notificationManager;

    public RemoveScreenSharingNotificationUseCase(INotificationManager notificationManager) {
        this.notificationManager = notificationManager;
    }

    public void execute() {
        this.notificationManager.removeScreenSharingNotification();
    }
}

File: widgetssdk/src/main/java/com/glia/widgets/core/notification/domain/RemoveScreenSharingNotificationUseCase.kt
package com.glia.widgets.core.notification.domain

import com.glia.widgets.core.notification.device.INotificationManager

internal class RemoveScreenSharingNotificationUseCase(private val notificationManager: INotificationManager) {
    operator fun invoke() = notificationManager.removeScreenSharingNotification()
}